samples/client/petstore/dart2/petstore_client_lib/lib/api/store_api.dart

```diff
@@ -90,41 +100,42 @@ class StoreApi {
     if(nullableContentType != null && nullableContentType.startsWith("multipart/form-data")) {
       bool hasFields = false;
       MultipartRequest mp = MultipartRequest(null, null);
-      if(hasFields)
+      if(hasFields) {
         postBody = mp;
-    }
-    else {
+      }
+    } else {
     }
 
-    var response = await apiClient.invokeAPI(path,
-                                             'GET',
-                                             queryParams,
-                                             postBody,
-                                             headerParams,
-                                             formParams,
-                                             nullableContentType,
-                                             authNames);
-    return response;
+    return await apiClient.invokeAPI(
+      path,
+      "GET",
+      queryParams,
+      postBody,
+      headerParams,
+      formParams,
+      nullableContentType,
+      authNames,
+    );
   }
 
   /// Returns pet inventories by status
   ///
   /// Returns a map of status codes to quantities
   Future<Map<String, int>> getInventory() async {
-    Response response = await getInventoryWithHttpInfo();
+    final response = await getInventoryWithHttpInfo();
     if(response.statusCode >= 400) {
       throw ApiException(response.statusCode, _decodeBodyBytes(response));
-    } else if(response.body != null) {
-      return Map<String, int>.from(apiClient.deserialize(_decodeBodyBytes(response), 'Map<String, int>'));
+    }
+    if(response.body != null) {
+      return Map<String, int>.from(apiClient.deserialize(_decodeBodyBytes(response), "Map<String, int>"));
           ;
```

Comment:
Do you got an idea why there is the lonely semicolon?

Reply:
I'll look into it :)

Reply:
Fixed ;)